src/Schema/Building/SchemaBuilderInterface.php

```diff
@@ -19,14 +19,4 @@ public function build(
         ResolverRegistryInterface $resolverRegistry,
         array $options = []
     ): Schema;
-
-    /**
-     * @param DocumentNode              $document
-     * @param ResolverRegistryInterface $resolverRegistry
-     * @return BuildingContextInterface
-     */
-    public function createContext(
```

Comment:
Remove this method from the interface because It doesn't have to be a part of the interface.